build.sh

```diff
@@ -104,11 +108,18 @@ fi
 # Build and install the cudf Python package
 if (( ${NUMARGS} == 0 )) || hasArg cudf; then
 
-    cd ${REPODIR}/python
+    cd ${REPODIR}/python/cudf
     if [[ ${INSTALL_TARGET} != "" ]]; then
 	python setup.py build_ext --inplace
 	python setup.py install --single-version-externally-managed --record=record.txt
     else
 	python setup.py build_ext --inplace --library-dir=${LIBCUDF_BUILD_DIR}
     fi
 fi
+
+# Build and install the dask_cudf Python package
+if (( ${NUMARGS} == 0 )) || hasArg dask_cudf; then
+
+    cd ${REPODIR}/python/dask_cudf
+	python setup.py install --single-version-externally-managed --record=record.txt
```

Comment:
Fix spacing here. Also remove the newline.

Reply:
Looks like the file had mixed tabs and spaces, fixing now.